src/Components/CreateImageWizardV2/steps/Oscap/Oscap.tsx

```diff
@@ -0,0 +1,274 @@
+    selection: DistributionProfileItem
+  ) => {
+    dispatch(changeOscapProfile(selection));
+    setIsOpen(false);
+  };
+
+  const options = [
+    <OScapNoneOption setProfileName={setProfileName} key="oscap-none-option" />,
+  ];
+  if (isSuccess) {
+    options.concat(
+      profiles.map((profile_id) => {
+        return (
+          <OScapSelectOption
+            key={profile_id}
+            profile_id={profile_id}
+            setProfileName={setProfileName}
+          />
+        );
+      })
+    );
+  }
+
+  if (isFetching) {
+    options.push(
+      <SelectOption
+        isNoResultsOption={true}
+        data-testid="policies-loading"
+        key={'None'}
+      >
+        <Spinner size="md" />
+      </SelectOption>
+    );
+  }
+
+  return (
+    <FormGroup
+      isRequired={true}
+      data-testid="profiles-form-group"
+      label={
+        <>
+          OpenSCAP profile
+          <Popover
+            maxWidth="30rem"
+            position="left"
+            bodyContent={
+              <TextContent>
+                <Text>
+                  To run a manual compliance scan in OpenSCAP, download this
+                  image.
+                </Text>
+              </TextContent>
+            }
+          >
+            <Button variant="plain" aria-label="About OpenSCAP" isInline>
+              <HelpIcon />
+            </Button>
+          </Popover>
+        </>
+      }
+    >
+      <Select
+        ouiaId="profileSelect"
+        variant={SelectVariant.typeahead}
+        onToggle={handleToggle}
+        onSelect={handleSelect}
+        onClear={handleClear}
+        selections={profileName}
+        isOpen={isOpen}
+        placeholderText="Select a profile"
+        typeAheadAriaLabel="Select a profile"
+        isDisabled={!isSuccess}
+        onFilter={(_event, value) => {
+          if (profiles) {
+            return [
+              <OScapNoneOption
+                setProfileName={setProfileName}
+                key="oscap-none-option"
+              />,
+            ].concat(
+              profiles.map((profile_id, index) => {
+                return (
+                  <OScapSelectOption
+                    key={index}
+                    profile_id={profile_id}
+                    setProfileName={setProfileName}
+                    input={value}
+                  />
+                );
+              })
+            );
+          }
+        }}
+      >
+        {options}
+      </Select>
+      {isError && (
+        <Alert
+          title="Error fetching the profiles"
+          variant="danger"
+          isPlain
+          isInline
+        >
+          Cannot get the list of profiles
+        </Alert>
+      )}
+    </FormGroup>
+  );
+};
+
+type OScapNoneOptionPropType = {
+  setProfileName: (name: string) => void;
+};
+
+const OScapNoneOption = ({ setProfileName }: OScapNoneOptionPropType) => {
+  return (
+    <SelectOption
+      value={undefined}
```

Comment:
Hmm, okay so value of the "None" option is undefined. This is just an idea, but does it mean that when we clear the Select it should display "None" instead of being empty?